modules/servers/openprovider/lib/openprovider_api.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'response.php';
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'last_request.php';
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'command_mapping.php';
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'api_configuration.php';
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'params_builder/params_creator_factory.php';
+
+use Openprovider\Api\Rest\Client\Base\Configuration;
+use GuzzleHttp6\Client as HttpClient;
+use Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer;
+
+class OpenProviderApi
+{
+    const API_CLIENT_NAME = 'whmcs-plesk-1';
+    const API_URL = 'https://api.openprovider.eu';
+    const API_CTE_URL = 'https://api.cte.openprovider.eu';
+
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    /**
+     * @var HttpClient
+     */
+    private $http_client;
```

Comment:
why  underscore?

Reply:
I have taken this client from blesta where we need to use underscore style.
We have already talked about code style with you and decided that each file or function can be with own style, so i don't think it's wrong